openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -664,12 +671,16 @@ def _gather(
                 rep = pu.outputs['repeat_id']
                 gen = pu.outputs['generation']
 
-                repeats[rep].append((gen, pu.outputs['nc']))
+                repeats[rep].append((
+                    gen, pu.outputs['nc'],
+                    pu.outputs['last_checkpoint']))
         data = []
         for rep in sorted(repeats.items()):
             # then sort within a repeat according to generation
-            nc_files = [ncpath for gen, ncpath in sorted(rep)]
-            data.append(nc_files)
+            nc_paths = [ncpath for gen, ncpath, nc_check in sorted(rep[1])]
+            chk_files = [nc_check for gen, ncpath, nc_check in sorted(rep[1])]
+            data.append({'nc_paths': nc_paths,
```

Comment:
passing checkpoints through here because the analyzer needs it to done all its necessary checks